app/models/order.rb

```diff
@@ -3,10 +3,11 @@ class Order < ActiveRecord::Base
   has_many :chips, through: :chip_orders
   belongs_to :user
   validates :total_price, numericality: { greater_than: 0 }
-  scope :ordered,    -> { where status: 'Ordered' }
-  scope :paid,       -> { where status: 'Paid' }
-  scope :cancelled,  -> { where status: 'Cancelled' }
-  scope :completed,  -> { where status: 'Completed' }
+  scope :ordered,       -> { where status: 'Ordered' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.